.github/workflows/release.yml

```diff
@@ -17,6 +17,8 @@ jobs:
       -
         name: Checkout
         uses: actions/checkout@755da8c3cf115ac066823e79a1e1788f8940201b # https://github.com/actions/checkout/releases/tag/v3.2.0
+        with:
+          ssh-key: "${{ secrets.GIT_SSH_PRIVATE_KEY }}"
       -
         name: Set up Go
```

Comment:
Also just realising we should not need this, right?

Reply:
It might be best to check out and push with the same SSH key - another option is to tell git more explicitly to use the SSH key in the push step, rather than just writing the file to `.ssh`.

If you want to remove this, I'd test again afterwards just in case.

I think both bits are fine as they are and would be happy to merge and release.